errors.go

```diff
@@ -0,0 +1,5 @@
+package safecast
+
+import "errors"
+
+var ErrIntegerOverflow = errors.New("integer overflow")
```

Comment:
what about float32 overflows?

Reply:
wording is a pain indeed

But for now, it's OK. I'm only converting as integer, even it I accept float, so for now the wording is OK.

But I'll change it anyway to avoid renaming again later

Reply:
I commented accordingly, it should be clearer